private/buf/cmd/buf/command/push/push_test.go

```diff
@@ -230,12 +231,13 @@ func appRun(
 	return appcmd.Run(
 		context.Background(),
 		app.NewContainer(
-			ammendEnv(
+			amendEnv(
 				internaltesting.NewEnvFunc(t),
 				func(env map[string]string) map[string]string {
 					env["BUF_TOKEN"] = "invalid"
 					buftransport.SetDisableAPISubdomain(env)
 					injectConfig(t, appName, env)
+					env[string(buffeature.TamperProofing)] = "1"
```

Comment:
These tests assume tamper proofing is enabled.

Reply:
They do. Can we also do a trivial test that checks nil manifest when the feature is disabled?

Reply:
For the BSR [some tests for flag value both ON/OFF](https://github.com/bufbuild/core/pull/7168/files#diff-4ca5dc2d241550fdcafdfefd9a0d27d2285bdb575aea9ee9b6a5ee12312810f2R328-R330) proved useful, probably worth to do it here as well?